samples/nrf9160/modem_shell/Kconfig

```diff
@@ -70,6 +70,13 @@ config MOSH_FOTA
 	help
 	  FOTA for performing software updates over-the-air for both modem and application side.
 
+config MOSH_REST_CLIENT
```

Comment:
Should we shorten this to the command name as for other commands so it would be just `MOSH_REST`
Sorry, I didn't realize this in my earlier comments while focusing still on the "Simple REST" concept.

Reply:
Changed.